installer/manifests/keptn/charts/control-plane/templates/_helpers.tpl

```diff
@@ -263,4 +260,17 @@ securityContext:
 {{- end -}}
 {{- end -}}
 
+{{- define "control-plane.common.update-strategy" -}}
+{{- if (.Values.common).strategy -}}
+strategy:
+{{- toYaml .Values.common.strategy | nindent 4 -}}
+{{- else -}}
+strategy:
+  type: RollingUpdate
+  rollingUpdate:
+    maxSurge: 1
+    maxUnavailable: 1
```

Comment:
the configuration here is different from the configuration in `values.yaml`. is this intended?